docs/integrate/dbeaver/index.md

```diff
@@ -0,0 +1,98 @@
+(dbeaver)=
+# DBeaver
+
+```{div}
+:style: "float: right; margin-left: 0.5em"
+[![](https://upload.wikimedia.org/wikipedia/commons/thumb/b/b5/DBeaver_logo.svg/512px-DBeaver_logo.svg.png){w=120px}](https://dbeaver.io/)
+```
+
+[DBeaver] is a multipurpose cross-platform database tool for developers,
+database administrators, analysts, and everyone working with data.
+
+It is available as an open-source version _DBeaver Community_ and
+as a commercial version _DBeaver PRO_.
+
+
+## Install
+
+::::{grid} 2
+:::{grid-item}
+For connecting to CrateDB, the [CrateDB JDBC Driver] will be used.
+:::
+:::{grid-item}
+![Image](https://github.com/user-attachments/assets/ebd11a53-4d4a-4a9b-bed2-25909e618929){w=480px}
+:::
+::::
+
+
+## Connect
+
+::::{grid} 2
+
+:::{grid-item}
+Please specify database URL and credentials of your CrateDB cluster.
+
+When connecting to [CrateDB Self-Managed] on localhost,
+for evaluation purposes, use:
+```
+jdbc:crate://localhost:5432/
+```
+
+When connecting to [CrateDB Cloud], use:
+```
+jdbc:crate://<clustername>.cratedb.net:5432/
+```
+:::
+:::{grid-item}
+![Image](https://github.com/user-attachments/assets/3288ab3f-a70a-42db-8b99-051ea3051c84){w=480px}
+:::
+:::{grid-item}
+When selecting CrateDB, drivers will be downloaded automatically
+on the first use.
+
+Alternatively, download and use the [crate-jdbc-standalone-2.7.0.jar] JAR file,
+and select the driver class `io.crate.client.jdbc.CrateDriver`.
+:::
+:::{grid-item}
+![Image](https://github.com/user-attachments/assets/5436f893-06d8-433f-94b3-72dddb5a13e1){w=480px}
+:::
+
+::::
```

Comment:
none of this is correct

Reply:
All of the information has been derived from the blog, in order to set the stage for doing it better here.

- https://cratedb.com/blog/cratedb-dbeaver

The idea is to come to the status of a canonical page [like the one for DataGrip](https://cratedb.com/docs/guide/integrate/datagrip/), that will include all the correct information for optimal usability.

Reply:
We need to do at least two things here:

1. @hlcianfagna and @hammerhead mentioned at https://github.com/crate/cratedb-guide/pull/155 that the screenshots are all wrong and outdated. This should probably be refreshed.
2. More detailed information about using DBeaver with CrateDB has been provided by @hlcianfagna at https://github.com/crate/crate-clients-tools/issues/206#issuecomment-2535188656. This should probably be added to the content of this page, other than commandeering it back to use the built-in integration for CrateDB?

Please let us know about anything else that you would like to see on this page, or send patches.

Reply:
That blog post is fairly outdated. There is a new CrateDB Connection in DBeaver for more than a year
https://github.com/dbeaver/dbeaver/pull/20549

I am surprised there is no blog / community post about it.

Reply:
> That blog post is fairly outdated. I am surprised there is no blog / community post about it.

I am not surprised. Around that time, many things started falling apart, stronger than before.

My intentions have been to wrap up the current status coming from https://github.com/dbeaver/dbeaver/pull/20549 into a concise documentation page on our realms, [like the one for DataGrip](https://cratedb.com/docs/guide/integrate/datagrip/), which also was long overdue.

I didn't expect this would raise any significant attention, so I am extra happy you are joining here.

Reply:
Now, let's wrap it up into this document, if you agree? At GH-155, admonitions from @hlcianfagna and @hammerhead have mostly been about screenshots. Now, we see it is also about content, which might not be much, because everything works out of the box?

So, what do you suggest? Just pick that single screenshot you added to the main PR description at https://github.com/dbeaver/dbeaver/pull/20549 into the page, and call it a day?

If you like it, maybe you can share two more screenshots with me that reflect something along the lines of the [DataGrip » Usage](https://cratedb.com/docs/guide/integrate/datagrip/#usage) section, i.e. displaying a tree and a table, possibly showing an excerpt of `sys.summits`?

Reply:
I've just updated the patch based on a few suggestions floating around.